task/test-tests-scan

```diff
@@ -206,6 +216,26 @@ class TestTestsScan(unittest.TestCase):
         self.assertEqual(output.strip(), expected_output)
         self.assertIsNone(stderr)
 
+    def test_tests_invoke(self):
+        repo = "cockpit-project/cockpit"
+        args = ["--revision", self.revision, "--repo", repo]
+        script = os.path.join(BOTS_DIR, "tests-invoke")
+        with tempfile.TemporaryDirectory() as tempdir:
+            testdir = f"{tempdir}/.cockpit-ci"
+            os.makedirs(testdir)
+            with open(f"{testdir}/run", "w") as fp:
+                fp.write("#!/bin/bash\nexit 1")
+            os.system(f"chmod +x {testdir}/run")
+            proc = subprocess.Popen([script, *args], stdout=subprocess.PIPE, universal_newlines=True,
+                                    env={"GITHUB_API": f"http://{ADDRESS[0]}:{ADDRESS[1]}"}, cwd=tempdir)
+            output, stderr = proc.communicate()
+            api = github.GitHub(f"http://{ADDRESS[0]}:{ADDRESS[1]}/")
+
+            self.assertEqual(output, "")
+            self.assertEqual(proc.returncode, 1)
+            self.assertEqual(len(api.get("issues")), 1)
```

Comment:
Could this check the content of the issue? that the expected SHA made it into the body? Or doesn't our mock server go that far?